resources/views/form/partials/tabs.twig

```diff
@@ -2,7 +2,7 @@
     {% spaceless %}
         {% for slug, tab in tabs %}
             <li class="nav-item {{ loop.first ? ' active' }}">
-                <a href="#tab-{{ slug }}" data-toggle="tab" class="nav-link">
+                <a href="#tab-{{ tab.slug ? tab.slug : slug }}" data-toggle="tab" class="nav-link">
```

Comment:
`<a href="#tab-{{ tab.slug|default(slug) }}" data-toggle="tab" class="nav-link">`

Reply:
Both acceptable, just depends how much Twig functionality we want to use. Not sure what would actually be quicker to render, @RyanThompson might know.

Reply:
`{{ tab.slug ?: slug }}` should work too...